spa/package.json

```diff
@@ -2,14 +2,25 @@
   "name": "spa",
   "private": true,
   "version": "0.0.0",
-  "type": "module",
+  "homepage": "/spa",
   "scripts": {
-    "dev": "vite",
-    "build": "tsc && vite build",
+    "dev": "BROWSER=none WDS_SOCKET_HOST=localhost WDS_SOCKET_PORT=3000 react-app-rewired start",
+    "build": "tsc && react-app-rewired build",
     "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
-    "preview": "vite preview",
     "test": "jest"
   },
+  "browserslist": {
+    "production": [
```

Comment:
Definitely need some discussion later. It is always a trade off of bundle size vs browser compatibility